src/main/scala/com/databricks/labs/overwatch/utils/Tools.scala

```diff
@@ -436,6 +437,44 @@ object Helpers extends SparkSessionWrapper {
     spark.conf.set("spark.databricks.delta.vacuum.parallelDelete.enabled", "false")
     s"SHRED COMPLETE: ${target.tableFullName}"
   }
+  /**
+   * Execute a parallelized streaming to follow the instructions provided through CloneDetail class
+   *
+   * @param cloneDetails details required to execute the parallelized clone
+   * @return
+   */
+  def tableStream(cloneDetails: Seq[CloneDetail]): Seq[CloneReport] = {
```

Comment:
this should be private[overwatch] I think since I don't think we want to allow use outside.

let's rename the func to make it more clear that it's only a snapStream.

I'm on the fence as to whether we should have a class/object called snapshot and refactor the core logic out of this Helpers section. We could move snap and this new `snapStream` over to the other location. We could have a Helper function that then instantiates/calls the core logic from Snapshot.scala or something. What do you think?